lib/Sae.js

```diff
@@ -93,41 +96,54 @@ export default class Sae extends BaseInput {
           value={value}
           onBlur={this._onBlur}
           onChange={this._onChange}
-          onFocus={this._onFocus}
+          onFocus={event => {
+            this.useNativeDriver = this.props.useNativeDriver;
+            this._onFocus(event);
+          }}
           underlineColorAndroid={'transparent'}
         />
-        <AnimatedIcon
-          name={iconName}
-          color={iconColor}
+        <Animated.View
           style={{
             position: 'absolute',
             bottom: 0,
-            right: focusedAnim.interpolate({
-              inputRange: [0, 1],
-              outputRange: [0, width + PADDING],
-            }),
+            right: 0,
             transform: [{
-              rotate: focusedAnim.interpolate({
+              translateX: width !== undefined ? focusedAnim.interpolate({
                 inputRange: [0, 1],
-                outputRange: ['0deg', '-90deg'],
-              }),
-            }],
-            fontSize: 20,
-            backgroundColor: 'transparent',
+                outputRange: [0, -(width + PADDING)]
+              }) : 0
+            }]
           }}
-        />
+        >
+          <AnimatedIcon
+            name={iconName}
+            color={iconColor}
+            style={{
+              transform: [{
+                rotate: focusedAnim.interpolate({
+                  inputRange: [0, 1],
+                  outputRange: ['0deg', '-90deg']
+                })
+              }],
+              fontSize: 20,
+            }}
+          />
+        </Animated.View>
         {/* bottom border */}
         <Animated.View
           style={{
+            width,
             position: 'absolute',
             bottom: 0,
             right: 0,
             height: 2,
-            width: focusedAnim.interpolate({
-              inputRange: [0, 1],
-              outputRange: [0, width],
-            }),
-            backgroundColor: iconColor,
+            transform: [{
+              rotateY: focusedAnim.interpolate({
```

Comment:
It changes a little bit the behavior, I couldn't reproduce exactly the same effect because native drivers doesn't support the width property